woocommerce/woocommerce-android
# [Tablet Orders] Mapping from the order to the new vm

Closes: #10472


### Description

* Adds mapping of the order/viewstate date to the new view models displayed on the drawer
* Adjusted the usage of the FF so we map the payments section but just hide that if FF is on
* Small UI tweaks

### Testing instructions

* Add Products, Custom Amount, Shipping, Coupons, Gift Card -> Notice that all are displayed correctly -> Verify that buttons for shipping/coupons git card work
* Try the same in the edit mode
* Make sure that there are no changes in the release mode

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/a66955db-1c61-456d-b949-12244a74b91f
